content/doc/book/managing/nodes.adoc

```diff
@@ -26,65 +24,44 @@ https://github.com/jenkins-infra/jenkins.io/pull/4612 is merged
 
 Jenkins controller::
 
-The Jenkins controller is the Jenkins service itself
-and is where Jenkins is installed.
-It is a webserver that also acts as a "brain"
-for deciding how, when and where to run tasks.
-Management tasks (configuration, authorization, and authentication)
-are executed on the controller, which serves HTTP requests.
-Files written when a Pipeline executes are written to the filesystem on the controller
-unless they are off-loaded to an artifact repository such as Nexus or Artifactory.
+The Jenkins controller is the Jenkins service itself and is where Jenkins is installed.
+It is a web server that also acts as a "brain" for deciding how, when, and where to run tasks. 
+Management tasks such as configuration, authorization, and authentication are executed on the controller, which serves HTTP requests. 
+Files written when a Pipeline executes are written to the filesystem on the controller, unless they are off-loaded to an artifact repository such as Nexus or Artifactory.
 
 Nodes::
 
-Nodes are the "machines" on which build agents run.
-Jenkins monitors each attached node for
-disk space, free temp space, free swap,
-clock time/sync and response time.
-A node is taken offline if any of these values
-go outside the configured threshold.
-
-Jenkins supports two types of nodes: _agents_ (described below) and the _built-in node_.
-The built-in node is a node that exists within the controller process.
-Both agents and the built-in node can be used to run tasks.
-However, running tasks on the built-in node is discouraged for security, performance, and scalability reasons.
-The number of executors configured for the node determines the node's ability to run tasks.
+Nodes are the "machines" on which build agents run. 
+Jenkins monitors each attached node for disk space, free temp space, free swap, clock time/sync, and response time. 
+A node is taken offline if any of these values go outside the configured threshold.
+Jenkins supports two types of nodes: *agents*, which are described below, and the *built-in node*. 
+The built-in node is a node that exists within the controller process. 
+Both agents and the built-in node can be used to run tasks. 
+However, running tasks on the built-in node is discouraged for security, performance, and scalability reasons. 
+The number of executors configured for the node determines the node's ability to run tasks. 
```

Comment:
are the ** in line 40 supposed to mark it as an important point @kmartens27 ?

Reply:
Hi @Darshan174, so the double ** is actually used to indent the text so that it is clearly part of the built-in node description. This is the reference I'm using in this case: https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/#nested-unordered-list

Reply:
Thanks for that reference @kmartens27.